### PR TITLE
frontend: fix null when CatalogSource not available

### PR DESCRIPTION
**Pull request by suomiy**

### CODE REVIEW

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by coreos-ui** [comment]
Can one of the admins verify this patch?

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/cloud-services/catalog-source.tsx [hunk 1]
```diff
@@ -154,7 +154,7 @@ export const CatalogSourceList = withFallback((props: CatalogSourceListProps) =>
 
 export const CatalogSourcesPage: React.SFC<CatalogSourcePageProps> = (props) => {
   type Flatten = (resources: {[kind: string]: {data: K8sResourceKind[]}}) => K8sResourceKind[];
-  const flatten: Flatten = resources => resources.catalogSource.data;
+  const flatten: Flatten = resources => resources.catalogSource ? resources.catalogSource.data : [];
```
Comment (jhadvig) [inline frontend/public/components/cloud-services/catalog-source.tsx]: Can be just
```js
_.get(resources.catalogSource, 'data', [])
```

**Pull request merged**